content/spin/v2/spinkube.md

```diff
@@ -0,0 +1,22 @@
+title = "SpinKube: Spin on Kubernetes"
```

Comment:
suggest 

```suggestion
title = "Spin on Kubernetes"
```

because this is kind of a roadmap to SpinKube, FP4K, the old container way, etc. - it's a great landing page and where we want people to come in

Reply:
Thanks @itowlson 
`Spin on Kubernetes` it is.